### DISCUSSION TITLE
Dr. Tass's new haptic vibrator

### DISCUSSION THREAD

**Discussion by TactileDesign** [kriswilk/buzzah#19] [Q&A]
During one of Dr. Tass's Q/A sessions on Youtube, he mentioned that his team had switched designs to incorporate a new haptic fingertip buzzer that costs only a "few dollars".  Does anyone know what haptic Dr. Tass switched to???

**Comment by Neurosmith24** [comment]
I saw that as well. But I don't think he has said, so only he and the design team probably know. There's been speculation that it _might_ be an audio exciter (speculated by someone who made gloves using audio exciters). But, that's just conjecture.

**Reply by TactileDesign** [reply]
Hi Jeff [Neurosmith24](https://github.com/Neurosmith24),  Thanks for your help!  I have been reading through discussions here and noticed that you had been corresponding with Phillip [PWPInnovator898](https://github.com/PWPInnovator898) who had amazingly tried out an actual Tass glove and was able to subjectively compare vibrational amplitudes to the Buzzah he had built for himself.  Phillip appeared to recommend the VG1040003D LRA over the ‎VG0832022D‎ as being closer to the amplitude of the Tass glove transducers.  This information is so helpful!  I am curious if you ended up using the VG1040003D in your prototypes?  Also, could you compare/contrast the feel of one of the LRA's you ended up using within and without your tactor housing?  Is the vibration that much different with the housing?  Thanks so much!  I appreciate your help and am looking forward to getting my parts next week!  Mark  [TactileDesign](https://github.com/TactileDesign)

**Reply by Neurosmith24** [reply]
I ended up using the VLV101040A - https://www.vybronics.com/linear-lra-vibration-motors/v-lv101040a also based on Phillip's recommendation in one of the discussions here. I think they end up buzzing the same whether they're within a housing or not. The issue comes in trying to focus the vibration on a 5mm area of the finger tip. I think the housings have helped with that to some degree. I've also incorporated foam padding in the housings to try to dampen vibrations carrying through to the housing itself. It's been moderately successful in helping to focus the vibration.
